pa11ycrawler/templates/index.html

```diff
@@ -18,8 +18,8 @@ <h2 class="hd-4">Pages Checked</h2>
     data-search="true">
     <thead>
         <tr>
-            <th data-field='Page Title' data-sortable="true">Page Title</th>
-            <th data-field='Results Link' data-sortable="true">Results Link</th>
+            <th data-field='Page' data-sortable="true">Page</th>
```

Comment:
This is the heading for the column that has page titles in it. I think it's clearer to use "Page" or "Page Title", but it's up to you.